src/qt/walletcontroller.cpp

```diff
@@ -203,6 +203,7 @@ void WalletControllerActivity::showProgressDialog(const QString& label_text)
     assert(!m_progress_dialog);
     m_progress_dialog = new QProgressDialog(m_parent_widget);
 
+    m_progress_dialog->setWindowTitle(PACKAGE_NAME);
```

Comment:
Search for `setWindowTitle`, typically windows/dialogs use titles. I'd add a `title` argument to `showProgressDialog` and update all call sites, like:
```cpp
showProgressDialog(tr("Open Wallet"), tr("Opening Wallet <b>%1</b>…").arg(name.toHtmlEscaped()));
```
which uses same text as the menu item.

Reply:
Addressed in  58dca47. Thanks for the suggestion @promag.